test/fixtures/snapshots/snapshot-no-match.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Possible to not have these comments in here? Having some non-jest-specific api for snapshot testing will allow us to change the underlying implementation in the future, if we want to.

Reply:
Looking into it, but I don't believe so. I bet we could codemod the snapshot engine though if needed.